lib/razorpay/payment.rb

```diff
@@ -10,6 +10,46 @@ class Payment < Entity
     def self.request
       Razorpay::Request.new('payments')
     end
+    
+    def self.createRecurringPayment(data={})
+      request.post "create/recurring" , data
+    end
+
+    def self.createJsonPayment(data={})
+      request.post "create/json" , data
+    end
```

Comment:
Not adding for otp generate and otp submit? - https://razorpay.com/docs/payments/payment-gateway/s2s-integration/json/v2/#integration-steps

Reply:
Support for S2S redirect?

Reply:
@arjunmenon we are not adding otp & redirect api .